packages/manager/src/views/WebPropertyListPage/components/WebPropertyCard/WebPropertyCard.tsx

```diff
@@ -24,7 +24,7 @@ export const WebPropertyCard = ({ title, subtitle, children, footer, isSelected
       <Title headingLevel="h3" size="xl" className="capitalize">
         {title}
       </Title>
-      <Text component="small" className="pf-u-color-400">
+      <Text component="h1" style={{ fontSize: '20px', marginTop: '10px', color: '#F4C145' }}>
```

Comment:
Also one more suggestion, 

That yellow subtitle takes too much attention. Can you change to following style and check it out.

That is color black or a bit gray , a bit more smaller font and then make it bold. 

This would give it enough attention

Reply:
Yellow makes it too stand out from the title